app/javascript/data/advancedFilterOptions.js

```diff
@@ -234,28 +243,58 @@ export const advancedFilterOptions = [
     title: 'Enrolled (Relative Date)',
     description: 'Monitorees enrolled in system during specified date range (relative to the current date)',
     type: 'relative',
-    hasTimestamp: true
+    hasTimestamp: true,
   },
   {
     name: 'latest-report-relative',
     title: 'Latest Report (Relative Date)',
     description: 'Monitorees with latest report during specified date range (relative to the current date)',
     type: 'relative',
-    hasTimestamp: true
+    hasTimestamp: true,
   },
   {
     name: 'last-date-exposure-relative',
     title: 'Last Date of Exposure (Relative Date)',
     description: 'Monitorees who have a last date of exposure during specified date range (relative to the current date)',
     type: 'relative',
-    hasTimestamp: false
+    hasTimestamp: false,
   },
   {
     name: 'symptom-onset-relative',
     title: 'Symptom Onset (Relative Date)',
     description: 'Monitorees who have a symptom onset date during specified date range (relative to the current date)',
     type: 'relative',
-    hasTimestamp: false
+    hasTimestamp: false,
+  },
+
+  /* RELATIVE DATE FILTER OPTIONS */
+  {
+    name: 'enrolled-relative',
```

Comment:
is there a reason these were added? they already exist above.  @tstrass this is probably why you were seeing double relative dates on demo

Reply:
this was probably an accident when resolving merge conflicts during a rebase: https://github.com/SaraAlert/SaraAlert/pull/861/commits/0b00f9efd83684aa21526ddee93b433c20c8fb79#diff-9609a958c89218f92ffdf7d0c6635f81734d4951c7261a249248114a23fcd24dL280